features/include/pcl/features/impl/fpfh_omp.hpp

```diff
@@ -62,7 +62,8 @@ pcl::FPFHEstimationOMP<PointInT, PointNT, PointOutT>::computeFeature (PointCloud
     for (size_t idx = 0; idx < indices_->size (); ++idx)
     {
       int p_idx = (*indices_)[idx];
-      if (this->searchForNeighbors (p_idx, search_parameter_, nn_indices, nn_dists) == 0)
+      if (!isFinite((*input_)[p_idx]) ||
```

Comment:
Please add a space between function name and braces to match PCL style. Also in the hunk below.

Reply:
Hi Sergey, I have added a space after each of the two calls to isFinite.